### PR TITLE
feat: Adds build folder and yml files for pipeline

### PR DESCRIPTION
**Pull request by DerekRaven**
GitHub Issue: #

## Proposed Changes

- Feature 
- Build or CI related changes 

## What is the current behavior?

No release CI

## What is the new behavior?

CI added to support NuGet releases

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] ~Documentation has been added/updated~
- [ ] ~Automated Unit / Integration tests for the changes have been added/updated~
- [x] Contains **NO** breaking changes
- [ ] ~Updated the Changelog~
- [ ] ~Associated with an issue~

### CODE REVIEW

File: build/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - feature/*
+
+resources:
+   containers:
+     - container: windows
+       image: nventive/vs_build-tools:17.1.1
+
+variables:
+- name: NUGET_VERSION
+  value: 6.1.0
```
Comment (LProulx86) [inline build/azure-pipelines.yml:14]: Why already at 6.1 ?

Comment (DerekRaven) [reply build/azure-pipelines.yml:14]: Based everything off an existing pipeline we're using for another project. Do you think we should be referencing a different version than this?

File: build/gitversion.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDeployment
+next-version: 0.4.0
```
Comment (carlh98) [inline build/gitversion.yml:3]: Not sure we had a discussion on version numbers, not sure what's the procedure about it

**Review by carlh98 — APPROVED** [review]

File: build/azure-pipelines.yml (hunk 1 above)
Comment (carlh98) [reply build/azure-pipelines.yml:14]: With vs_build-tools:17.1.1, I think we need 6.1.0

**Pull request merged**